test/e2e-go/e2e_go_olm_test.go

```diff
@@ -63,6 +63,17 @@ var _ = Describe("Integrating Go Projects with OLM", func() {
 				Expect(err).NotTo(HaveOccurred())
 			}
 
+			By("running the operator bundle using `run bundle` command")
+			runBundleCmd := exec.Command(tc.BinaryName, "run", "bundle", bundleImage, "--namespace", tc.Kubectl.Namespace)
+			_, err = tc.Run(runBundleCmd)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("destroying the Operator deployed with the 'run' subcommand")
+			cleanupPkgManCmd := exec.Command(tc.BinaryName, "cleanup", projectName,
+				"--timeout", "4m")
```

Comment:
You are running the bundle in a specific namespace. The ns that. is created for the operator (project-system). 
Should not the cleanup be done with the ns as well?